drake/systems/cascade_system.h

```diff
@@ -0,0 +1,112 @@
+#ifndef DRAKE_SYSTEMS_CASCADE_SYSTEM_H_
```

Comment:
No intentional code changes - those are the next PR.  Someday, I hope test coverage will be good enough that this question doesn't have to be asked! :-)